scripts/scenarios/cloud6/qa-scenario-2b-sbd-kvm-vmware.yaml

```diff
@@ -131,6 +131,9 @@ proposals:
         host: vcs.qa.suse.de
         user: "@@vcenter_user@@"
         password: "@@vcenter_password@@"
+        cluster_name: []
+        insecure: false
```

Comment:
Changed to `true`.

As to whether the option is mandatory or not... I guess @mjura should know :-)